packages/components/tabs/src/use-tabs.ts

```diff
@@ -174,13 +181,18 @@ export function useTabs<T extends object>(originalProps: UseTabsProps<T>) {
   );
 
   const getTabListProps: PropGetter = useCallback(
-    (props) => ({
-      ref: domRef,
-      "data-slot": "tabList",
-      className: slots.tabList({class: clsx(classNames?.tabList, props?.className)}),
-      ...mergeProps(tabListProps, props),
-    }),
-    [domRef, tabListProps, classNames, slots],
+    (props) =>
+      mergeProps(
+        tabListProps,
+        {
+          ref: domRef,
+          "data-slot": "tabList",
+          "aria-orientation": orientation,
+          className: slots.tabList({class: clsx(classNames?.tabList, props?.className)}),
+        },
+        props,
+      ),
+    [domRef, tabListProps, classNames, slots, orientation],
```

Comment:
I don't think you need to update this part. `aria-orientation` is already included in `tabListProps`. Also don't see the reason why we need to wrap everything in `mergeProps`.